pkg/pdutil/placement_client.go

```diff
@@ -0,0 +1,192 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pdutil
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"path"
+	"time"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/log"
+	"github.com/pingcap/tidb/store/pdtypes"
+	"github.com/pingcap/tiflow/pkg/config"
+	"github.com/pingcap/tiflow/pkg/httputil"
+	"github.com/pingcap/tiflow/pkg/retry"
+	pd "github.com/tikv/pd/client"
+	"go.uber.org/zap"
+)
+
+const (
+	updateTypeDel = iota
+	updateTypeAdd
+)
+
+const (
+	// DefaultMaxRetry is the default max number of verifing placement rules.
+	DefaultMaxRetry   = 3
+	batchUpdatePrefix = "/pd/api/v1/config/rules/batch"
+	idQueryPrefix     = "/pd/api/v1/config/rule"
+
+	// Split the default rule by `6e000000000000000000f8` to keep metadata region
+	// isolated from the normal data area. Note that the default rules also need
+	// to be modified here to avoid rule stacking.
```

Comment:
```suggestion
	// to be modified here to avoid rule stacking.  Ref: https://docs.pingcap.com/tidb/stable/configure- 
        // placement-rules
```